### PR TITLE
work in progress, add search keyword update action

### PR DESCRIPTION
**Pull request by sebbulon**
WIP, added search keyword action

### CODE REVIEW

File: src/coffee/import.coffee [hunk 1]
```diff
@@ -261,12 +261,13 @@ class Import
...
         when 'setMetaDescription' then header.has(CONS.HEADER_META_DESCRIPTION) or header.hasLanguageForBaseAttribute(CONS.HEADER_META_DESCRIPTION)
         when 'setMetaKeywords' then header.has(CONS.HEADER_META_KEYWORDS) or header.hasLanguageForBaseAttribute(CONS.HEADER_META_KEYWORDS)
+        when 'setSearchKeywords' then header.has(CONS.HEADER_SEARCH_KEYWORDS) or header.hasLanguageForBaseAttribute(CONS.HEADER_SEARCH_KEYWORDS)
         when 'addToCategory', 'removeFromCategory' then header.has(CONS.HEADER_CATEGORIES)
         when 'setTaxCategory' then header.has(CONS.HEADER_TAX)
         when 'setSKU' then header.has(CONS.HEADER_SKU)
         when 'addVariant', 'addPrice', 'removePrice', 'changePrice', 'addExternalImage', 'removeImage' then true
         when 'removeVariant' then @allowRemovalOfVariants
-        else throw Error "The action '#{action.action}' is not supported. Please contact the SPHERE.IO team!"
+        else throw Error "The action '#{action.action}' is not supported. Please contact the commercetools support team!"
```
Comment (hajoeichler) [inline src/coffee/import.coffee]: :+1:

**Comment by sebbulon** [comment]
@PhilippSpo happy for me to merge this? Tests go green :)

**Comment by emmenko** [comment]
@sebbulon one question: search keywords can be defined with different tokenizers. Do you consider this in the implementation? Or how does that work with the csv import?

**Comment by sebbulon** [comment]
@emmenko indeed .. custom tokenisers are not supported right now by this PR

**Comment by emmenko** [comment]
So how is this going to work?

**Comment by PhilippSpo** [comment]
@sebbulon There are no tests for updating existing search keywords, right?

**Comment by sebbulon** [comment]
@emmenko I didn't really change anything there as you can see. The CSV import already supported search keyword export to some extend, I added Search Keyword import - there was only one little change necessary. 
When you create the product you need to select the tokeniser. 
If you create the product with this tool right now, it will use default tokeniser - which seems to be not white space, which is not ideal I have to admit.

I guess we need to make this clear in the documentation + have another change request to support the different tokenisers.

**Comment by sebbulon** [comment]
@PhilippSpo right tests are not complete

**Comment by sebbulon** [comment]
@emmenko to answer your question - the ususal case how people use this is to tokenise the search keywords already. If they are provided separated by semicolon (or whatever the secondary delimiter is), it will be imported into separate text elements, each of them being a different token. So they technically don't need a whitespace tokeniser. 
Of course we need to add the support later down the line though to support the complete feature.

I will add this to the readme

**Comment by PhilippSpo** [comment]
OK we can merge this in my opinion.
Later on we also need to add support for only updating search keywords of a specific language, since this is a frequent use case. The problem here is, that there is no update action for only updating the searchKeywords of one language. Thus we would need to enrich the action with the current search keywords before sending the update action..

**Pull request merged**